### ISSUE TITLE
Variable font with weight axis

### ISSUE THREAD

**Issue by yoshiask** [notofonts/coptic#15]
**Is your feature request related to a problem? Please describe.**
Many other Noto Sans fonts, such as Latin-Greek-Cyrillic, have a version with variable weights and widths. Noto Sans Coptic however has a single static font, containing only the Regular font weight. This makes NSC difficult to use alongside English and Greek text with bold and black runs.

**Describe the solution you'd like**
Ideally, a variable font would be created with at least a weight axis.

**Describe alternatives you've considered**
A static Bold font would be acceptable, though a variable font would be preferred to match compatibility with other Noto Sans fonts.

**Additional context**
N/A

**Comment by simoncozens** [comment]
I agree this would be a great idea - and it's something on our wishlist as we take Noto into the future. But it's quite a lot of work to get a new weight drawn, and it's unfortunately not currently something we can make a priority. I'll keep this issue open because we want to see it happen, but I don't have a timescale for you yet.
